elements/lisk-p2p/src/validation.ts

```diff
@@ -195,3 +197,86 @@ export const checkPeerCompatibility = (
 		success: true,
 	};
 };
+
+export interface PeerLists {
```

Comment:
Maybe we can add this interface in `p2p_types.ts` and reuse it in `P2PConfig` and also import it here?